apps/web/stores/AnswerStore.ts

```diff
@@ -0,0 +1,70 @@
+// 3rd party
+import { makeAutoObservable } from "mobx";
+// local
+import { WalkthroughRootStore } from "./WalkthroughRootStore";
+
+export type WalkthroughAnswerType = string | string[] | Record<string, string>;
+export type WalkthroughStoreAnswers = Record<string, WalkthroughAnswerType>;
+
+export const DEFAULT_ANSWER_VALUE_MULTI_CHOICE = undefined;
+export const DEFAULT_ANSWER_VALUE_MULTI_CHOICE_MULTI = [];
+
+export class AnswerStore {
+  rootStore: WalkthroughRootStore;
+
+  answers: WalkthroughStoreAnswers = {};
+
+  constructor(rootStore: WalkthroughRootStore) {
+    makeAutoObservable(this);
+    this.rootStore = rootStore;
+  }
+
+  setAnswerValue = (value: WalkthroughAnswerType, questionId: string) => {
+    // if current question is NOT last one in questionHistory, remove all questions after current question and their answers
+    const currentQuestionIndex =
```

Comment:
Thinking if we add this it will make things more readable
` const { navigationStore: { questionHistory }, currentItemId } = this.rootStore;`